modules/cache2k/src/main/scala/scalacache/cache2k/Cache2kCache.scala

```diff
@@ -15,9 +12,7 @@ import scalacache.{AbstractCache, CacheConfig, Entry, Mode}
  *
  * This cache implementation is synchronous.
  */
-class Cache2kCache[V](underlying: CCache[String, Entry[V]])(implicit val config: CacheConfig,
-                                                            clock: Clock = Clock.systemUTC())
-    extends AbstractCache[V] {
+class Cache2kCache[V](underlying: CCache[String, Entry[V]])(implicit val config: CacheConfig) extends AbstractCache[V] {
```

Comment:
`Entry` is still needed to avoid runtime error

```
java.lang.IllegalArgumentException: The run time type is not available, got: V
  at org.cache2k.configuration.CacheTypeCapture.of(CacheTypeCapture.java:68)
  at org.cache2k.configuration.CacheTypeCapture.of(CacheTypeCapture.java:60)
  at org.cache2k.Cache2kBuilder.<init>(Cache2kBuilder.java:136)
  at scalacache.cache2k.Cache2kCache$$anon$1.<init>(Cache2kCache.scala:64)
  at scalacache.cache2k.Cache2kCache$.apply(Cache2kCache.scala:64)
  ... 36 elided
```